experimental/BLTUnfold/produceUnfoldingHistograms.py

```diff
@@ -223,7 +225,12 @@ def main():
                 response_only_fakes = Hist2D( bin_edges[variable], bin_edges[variable], name='response_only_fakes')      
 
                 if options.fineBinned:
-                    minVar = bin_edges[variable][0]
+                    if variable is 'HT':
+                        minVar = 120
+                    elif variable is 'ST':
+                        minVar = 146
+                    else:
+                        minVar = bin_edges[variable][0]
```

Comment:
I would undo this change since the ```bin_edges``` are included for the configuration (which is up-to-date).